src/components/dialogs/create-study-dialog/create-study-dialog-utils.js

```diff
@@ -5,15 +5,30 @@ import {
     DESCRIPTION,
     FORMATTED_CASE_PARAMETERS,
     STUDY_NAME,
+    ACTIVE_DIRECTORY,
 } from '../../utils/field-constants';
+import yup from '../../utils/yup-config';
 
-export const getCreateStudyDialogFormDefaultValues = () => {
+export const getCreateStudyDialogFormDefaultValues = ({
+    activeDirectory = '',
+}) => {
     return {
         [STUDY_NAME]: '',
         [DESCRIPTION]: '',
         [CASE_FILE]: null,
         [CASE_UUID]: '',
         [FORMATTED_CASE_PARAMETERS]: [],
         [CURRENT_PARAMETERS]: {},
+        [ACTIVE_DIRECTORY]: activeDirectory,
```

Comment:
You don't use redux anymore.
"`ACTIVE_DIRECTORY`" is meaningless here.
Call it just "`DIRECTORY`", it's the name you already use for the form in `handleCreateNewStudy`